pkgs/by-name/gh/gh-classroom/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule (finalAttrs: {
+  pname = "gh-classroom";
+  version = "0.1.14";
+
+  src = fetchFromGitHub {
+    owner = "github";
+    repo = "gh-classroom";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-h9j8B/MGZ4JJOJRj41IIQ9trQJZ4oqvT6ee9lc0P4oo=";
+  };
+
+  vendorHash = "sha256-UFV3KiRnefrdOwRsHQeo8mx8Z+sI1Rk5yu3jdZxUHxo=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${finalAttrs.version}"
+  ];
+
+  meta = {
+    homepage = "https://github.com/github/gh-classroom";
+    description = "gh-classroom is an Extension for the GitHub CLI, that enhances it for educators using GitHub classroom";
```

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
> Not start with the package name.
```suggestion
    description = "Extension for the GitHub CLI, that enhances it for educators using GitHub classroom";
```